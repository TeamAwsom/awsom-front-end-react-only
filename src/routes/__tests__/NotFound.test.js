import React from 'react';
import NotFound from '../NotFound';
import renderer from 'react-test-renderer';

describe('NotFound', () => {
  it('renders as expected', () => {
    const tree = renderer.create(<NotFound />).toJSON();

    console.log(tree);
    expect(tree).toMatchSnapshot();
  });
});
