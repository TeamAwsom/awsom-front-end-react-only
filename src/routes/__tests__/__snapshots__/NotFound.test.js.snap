// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders as expected 1`] = `
<div>
  <div
    className="container notfound"
  >
    <h1>
      404 - Not Found
    </h1>
  </div>
</div>
`;
